fabtests/common/check_hmem.c

```diff
@@ -44,7 +44,7 @@ int main(int argc, char** argv)
 		return EXIT_FAILURE;
 	hints->mode = ~0;
 	hints->domain_attr->mode = ~0;
-	hints->domain_attr->mr_mode = ~(FI_MR_BASIC | FI_MR_SCALABLE);
+	hints->domain_attr->mr_mode = FT_MR_ALL | FI_MR_HMEM;
```

Comment:
It's interesting that "ALL" doesn't actually mean "ALL"

Reply:
It's interesting that leaving somehow made you *more* annoying